owrx/source.py

```diff
@@ -378,15 +378,23 @@ def activateProfile(self, profile_id=None):
 
 class RtlSdrSource(SdrSource):
     def getCommand(self):
-        return "rtl_sdr -s {samp_rate} -f {center_freq} -p {ppm} -g {rf_gain} -"
+        command = "rtl_sdr -s {samp_rate} -f {center_freq} -p {ppm} -g {rf_gain}"
+        if self.rtlProps["device"] != "" :
```

Comment:
Again, it makes "device" not optional.
```suggestion
        if "device" in self.rtlProps:
```